src/PodiumdAdapter.Web/Endpoints/ObjectenEndpoints.cs

```diff
@@ -439,5 +439,33 @@ private static bool TryParseContactmomentId([NotNullWhen(true)] JsonNode? json,
 
             return !string.IsNullOrWhiteSpace(result);
         }
+
+        private static string UpdateToelichtingWithRecentsteVoorlopigAntwoord(JsonObject obj, JsonNode? contact)
+        {
+            var toelichtingNodeClone = obj["toelichting"]?.DeepClone();
```

Comment:
deepclone en dan toString? als toelichting niet een enkele string maar een jsonobject zou bevatten, wat zou dan de resultaat van deze functie zijn? Ik vermoed dat er voorheen op alle velden waar verder niets customs mee gedaan werd uit voorzorg een Deepclone gedaan werd. Ik denk dat je in deze situatie moet controleren of de toelichting zoals verwacht een string is en heeft Deepclone dan verder ook geen toegevoegde waarde

Reply:
ja die was ook redundant ik heb het eruit gehaald, we hebben alleen toelichting als text nodig.

Reply:
jij gebruikt .ToString elder wordt .GetValue<string> gebruikt. wat is het verschil?